<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>헬로 파이큐티</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>편집</string>
    </property>
    <addaction name="action_selectAll"/>
    <addaction name="action_cut"/>
    <addaction name="action_search"/>
   </widget>
   <widget class="QMenu" name="menu_form">
    <property name="title">
     <string>서식</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>보기</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>도움말</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_form"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save">
   <property name="text">
    <string>저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>열기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="text">
    <string>다른 이름으로 저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>종료</string>
   </property>
  </action>
  <action name="action_selectAll">
   <property name="text">
    <string>전체선택</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>잘라내기</string>
   </property>
  </action>
  <action name="action_search">
   <property name="text">
    <string>찾기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
